### ISSUE TITLE
(phi1002) brackets?

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/catalog_data#165]
We removed the brackets around phi1002 per this PR https://github.com/PerseusDL/canonical-latinLit/pull/291
I see that this breaks the sorting in the SDL and think that if this is a spurious attribution, we need a better indicator and should make one for all collections. Perhaps (sp.) or (dub.) after the author group?

**Comment by AlisonBabeu** [comment]
I agree. I've never had a standard cataloging practice on this I must admit but tended to follow the pattern used for an author work in the TLG/PHI typically, sometimes it was brackets, other times it is a (sp.) or a (dub.) and other times there was a Pseudo-Quintilian or other pseudo-author used.  I'm not entirely sure why pseudo is sometimes used and in other cases not. This definitely merits further thought and a decision made for consistency.
